instructure/canvas-android
# [MBL-16882][Student] HTML Parsing

Test plan:
- Verify that all the images are displayed correctly in HTML content (pages, assignment details, quiz details, syllabus, syllabus events)
- Verify that the above behavior is correct also when file sync is not checked.
- Check that the progress is correct on the offline sync progress screen.
- Verify that all the external and internal files are cleaned up when sync is disabled for a course.
- Discussions and announcements should work only after the discussion/announcement details ticket.

refs: MBL-16682
affects: Student
release note: none

## Checklist
